src/main/java/com/three_tech_solutions/slot_app/controllers/responses/PriceUpdateResponse.java

```diff
@@ -0,0 +1,14 @@
+package com.three_tech_solutions.slot_app.controllers.responses;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+import java.util.UUID;
+
+@Getter
+@AllArgsConstructor
+public class PriceUpdateResponse {
```

Comment:
Esto no seria un `PriceUpdateResponse`, sino que representa a la respuesta de un precio, le cambiaria el nombre a `PriceResponse`

```suggestion
public class PriceResponse {
```